examples/vm-alpine-datavolume.yaml

```diff
@@ -19,7 +19,7 @@ spec:
             storage: 2Gi
         storageClassName: local
       source:
-        http:
+        registry:
```

Comment:
AFAIK, the examples are generated. Probably that should be fixed at: https://github.com/kubevirt/kubevirt/blob/1bd0fb414531b432d1ee3438e9a57e49b773e5c2/tools/vms-generator/utils/utils.go#L990 and then `make generate`